src/golive/cdn.md

```diff
@@ -52,3 +52,51 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you generally don't want users to access your platform.sh project directly. Luckily, you can secure your connection between you and the CDN with [client authenticated TLS](/configuration/routes/https.html#client-authenticated-tls). 
```

Comment:
I'd use slightly different wording for this sentence: `When using a CDN, you might not want users to access your platform.sh origin directly.`